### PR TITLE
SLA - extended graphing - icmp-jitter

### PR DESCRIPTION
**Pull request by adaniels21487**
- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

- Added more graphs for rtt_type: icmpjitter

### CODE REVIEW

**Comment by adaniels21487** [comment]
Resolves #4341

File: includes/polling/cisco-sla.inc.php [hunk 1]
```diff
@@ -38,15 +38,15 @@
         $rtt = $rttMonLatestRttOperTable['1.3.6.1.4.1.9.9.42.1.2.10.1.1'][$sla_nr];
         echo 'SLA '.$sla_nr.': '.$rtt_type.' '.$sla['owner'].' '.$sla['tag'].'... '.$rtt.'ms at '.$time.'\n';
 
-        $fields = array(
+        $metrics = array(
```
Comment (laf) [inline includes/polling/cisco-sla.inc.php]: Probably best keeping this as `$fields` like it is everywhere else.

File: includes/polling/cisco-sla.inc.php [hunk 2]
```diff
@@ -82,6 +82,36 @@
                 data_update($device, 'sla', $tags, $jitter);
                 $metrics = array_merge($metrics, $jitter);
                 break;
+            case 'icmpjitter':
+                $icmpjitter = array(
```
Comment (laf) [inline includes/polling/cisco-sla.inc.php]: Same, this should be `$fields` to be consistent.

**Comment by laf** [comment]
image

Thanks @adaniels21487 :)

**Pull request merged**